### PR TITLE
Ros configs

### PR DESCRIPTION
**Pull request by CA-rvinson**
Added config file to control the color and thus intensity of the tracking and burning laser. The config follows the ros2 structure, which has a bit of seemingly unnecessary boilerplate, but should be stable and feature reach as the project grows.

### CODE REVIEW

File: laser_runner_removal/laser_runner_removal/main_node.py [hunk 1]
```diff
@@ -219,6 +228,11 @@ def execute(self, blackboard):
     def _correct_laser(self, laser_send_point, blackboard):
         """Iteratively move the tracking laser until it is at the same point as the runner."""
         blackboard.curr_track.corrected_point = laser_send_point
+        tracking_laser = (
```
Comment (genkikondo) [inline laser_runner_removal/laser_runner_removal/main_node.py:231]: we should centralize access to the config values

File: laser_runner_removal/laser_runner_removal/main_node.py [hunk 2]
```diff
@@ -314,6 +340,15 @@ def __init__(self):
         """State Machine that controls what the system is currently doing"""
         Node.__init__(self, "main_node")
 
+        self.declare_parameters(
+            namespace="",
+            parameters=[
```
Comment (genkikondo) [inline laser_runner_removal/laser_runner_removal/main_node.py:345]: are these defaults?

**Review by genkikondo — APPROVED** [review]
Looks great. Sorry for the merge conflict

File: laser_runner_removal/laser_runner_removal/main_node.py (hunk 1 above)
Comment (CA-rvinson) [reply laser_runner_removal/laser_runner_removal/main_node.py:231]: Not really sure what you mean. Are you thinking of calling the all the config get functions a la. `tracking_laser=blackboard.main_node.get_parameter("tracking_laser").get_parameter_value().integer_array_value` in one place, and making attributes for the main node? I'm down.

File: laser_runner_removal/laser_runner_removal/main_node.py (hunk 2 above)
Comment (CA-rvinson) [reply laser_runner_removal/laser_runner_removal/main_node.py:345]: Yeah, I can add a comment

**Pull request merged**